src/sonic-eventd/tools/events_monit_test.py

```diff
@@ -0,0 +1,236 @@
+#! /usr/bin/python -u
```

Comment:
>#! /usr/bin/python [](http://example.com/codeflow?start=0&length=18)

Best practice is to use `#!/usr/bin/env python3`
ref: [https://stackoverflow.com/a/19305076/2514803](https://stackoverflow.com/a/19305076/2514803)

Reply:
Done

Reply:
remove blank `#!/usr/`

Reply:
Still applicable.

Reply:
We are using 

#!/usr/bin/env python3 as suggested.

Reply:
```diff
-#! /usr/bin/env python3
+#!/usr/bin/env python3
```

Reply:
Done